### PR TITLE
Print size of pcoinsTip cache in AcceptToMemoryPool

### PR DESCRIPTION
**Pull request by pstratem**
What the title says.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -4727,10 +4727,11 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             RelayTransaction(tx);
             vWorkQueue.push_back(inv.hash);
 
-            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u kB)\n",
+            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u MiB, cache=%.1fMiB(%utx))\n",
```
Comment (morcos) [inline src/main.cpp]: sorry to be annoying, but might be nice if they matched up..   maybe:
`LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %.1f MiB, cache %u txn, %.1f MiB)\n"`

Comment (MarcoFalke) [inline src/main.cpp]: Why `MiB`?

Comment (laanwj) [inline src/main.cpp]: Unfortunately, the other place that logs the cache size is MiB as well: https://github.com/bitcoin/bitcoin/blob/master/src/main.cpp#L2319

-1 on changing the mempool size to be in MiB as well though - note that fees are per 1000, not per 1024, so apart from SI standards adherence kB/MB makes more sense in the context.

Comment (MarcoFalke) [inline src/main.cpp]: I thought we were parsing the maxmempool size in `MB` so it should be changed to `kB` in the other places instead.

**Comment by morcos** [comment]
Let's just make an executive decision on what places in the code will use MB and what will use MiB.  And then we can move towards that even if not everything is correct now.  Seems a bit silly to me to have two different prefix types sitting right next to each other, but if that's what we decide for some reason, then fine, but at least we'll have our justification and we won't have this debate again.  I don't care what we choose, just would like to know what I should use in the future.

As for scale, personally I think on the order of MB makes more sense b/c cache and mempools are typical on the order of 100 MB.  So 213231 is harder to read than 213.2 if you ask me.

**Comment by MarcoFalke** [comment]
> executive decision on what places in the code will use MB and what will use MiB

According to #6981 we are using MB, kB etc. As 0.12 is not yet released everyone is welcome to change newly introduced MiB, kiB (like maxuploadtarget) to SI units.

**Comment by morcos** [comment]
oh great, sorry i missed that.

**Comment by laanwj** [comment]
The preference is SI units everywhere, so MB, GB, kB which are powers of 1000. To deviate from that at least have a good reason (not "I like powers of two better").

Which power of 1000 should be used depends on the quantity in question. For reporting of memory sizes, I think usually kB is the best choice not MB, as to be able to see small changes. Disk sizes should be MB or GB.

**Comment by morcos** [comment]
Then i suggest we use kB for both mempool and cache size in this line regardless of the fact that the cache size is powers of 2 other places..   we can fix that later if needed...   lets move towards consistency

**Comment by MarcoFalke** [comment]
> we can fix that later

This is already a cleanup PR in some sense so no need to split it into two.

**Comment by MarcoFalke** [comment]
@pstratem Are you still working on this?

**Pull request closed without merging**